tee-worker/client-api/parachain-api/prepare-build/interfaces/omniExecutor/definitions.ts

```diff
@@ -12,7 +12,7 @@ export default {
         MrEnclave: "H256",
         NativeCall: {
             _enum: {
-                request_auth_token: "(LitentryIdentity, AuthOptions)",
```

Comment:
Can we release a new client-sdk in this PR? Then I will update it in the sandbox to adapt (remove the JWT expiration time parameter).

Reply:
Actually, let me revert the changes in the identity client-sdk, this should only affect the omni-executor. When the omni-executor's client sdk is ready we can adapt the demo app